sass/styles.scss

```diff
@@ -0,0 +1,155 @@
+//variables
+//$copper: #ba3c2c;
+$copper: #f93624;
+//$cobalt:#4d81d5;
+$butter: #fcde9c;
+$graphite: #262626;
+$roboto: "Roboto Condensed",
+arial,
+sans-serif;
+
+* {
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    background-color: #2e4dff;
+}
+
+/* ---------- Background Gradient ---------- */
+.ej2 {
+    
+    background-image: linear-gradient(#2e4dff, #ce5cff, #2e4dff, #B274D6, #2e4dff);
+    position: absolute;
+    height: 4000px;
+    left: 50%;
+    margin: 0 auto;
+    top: 50%;
+    transform: translate(-50%, -50%);
+    width: 480px;
+    z-index: 0;
+}
+
+/* ---------- Main Flex Container ---------- */
+.flex-container {
+    display: flex;
+    font-family: $roboto;
```

Comment:
Since you only use one font, consider moving this rule up to your `body` element.